contracts/services/IBorrowService.sol

```diff
@@ -3,7 +3,19 @@ pragma solidity ^0.8.16;
 import "./IService.sol";
 import "../smartwallet/IForwarder.sol";
 
+/**
+ * @title Borrow Service Interface
+ * @dev Interface for the Borrow Service contract
+ * @author RIF protocols team
+ */
 interface IBorrowService {
+    /**
+     * @notice Emitted when a new borrow service is consumed
+     * @param listingId The id of the listing
+     * @param borrower The address of the borrower
+     * @param currency The address of the currency
+     * @param amount The amount of the loan
+     */
     event Borrow(
```

Comment:
Duration is missing in the NatSpec comment.

Reply:
good catch